### PR TITLE
naming schema: couchbase

### PR DESCRIPTION
**Pull request by amarziali**
# What Does This Do

v1 naming schema changes for couchbase:
* service: `{{DD_SERVICE}}-couchbase`
* operation: `couchbase.query`

# Motivation

# Additional Notes

### CODE REVIEW

File: dd-java-agent/instrumentation/couchbase/couchbase-2.0/src/test/groovy/util/AbstractCouchbaseTest.groovy [hunk 1]
```diff
@@ -158,4 +158,19 @@ abstract class AbstractCouchbaseTest extends AgentTestRunner {
...
+
+  @Override
+  protected String service() {
+    return "couchbase"
+  }
+
+  @Override
+  protected String operation() {
+    return "couchbase.call"
+  }
```
Comment (PerfectSlayer) [inline dd-java-agent/instrumentation/couchbase/couchbase-2.0/src/test/groovy/util/AbstractCouchbaseTest.groovy:175]: Why settings default values with v0 name logic here?
I found clearer when v0 values are defined in child classes, near the v1 child classes so you can quickly see the behavior difference. (like you did in all other versioned tests)

File: dd-java-agent/instrumentation/couchbase/couchbase-2.0/src/test/groovy/CouchbaseClientTest.groovy [hunk 2]
```diff
@@ -123,3 +124,24 @@ class CouchbaseClientTest extends AbstractCouchbaseTest {
     cleanupCluster(cluster, environment)
   }
 }
+
+class CouchbaseClientV0ForkedTest extends CouchbaseClientTest {
+
```
Comment (PerfectSlayer) [inline dd-java-agent/instrumentation/couchbase/couchbase-2.0/src/test/groovy/CouchbaseClientTest.groovy:129]: So here, you miss the context of the values you are testing 😞

**Review by PerfectSlayer — APPROVED** [review]
Minor comment about one test case. LGTM otherwise 👍

File: dd-java-agent/instrumentation/couchbase/couchbase-2.0/src/test/groovy/util/AbstractCouchbaseTest.groovy (hunk 1 above)
Comment (amarziali) [reply dd-java-agent/instrumentation/couchbase/couchbase-2.0/src/test/groovy/util/AbstractCouchbaseTest.groovy:175]: I did this way not to check v0 and v1 on _every_ test. In this way I have the coverage for v1 that I need without changing all the tests. In this way we also ensure that, without changes, v0 is the one used

**Pull request merged**